cmd/gaia/testnets/README.md

```diff
@@ -88,194 +99,200 @@ Your full node has been cleanly upgraded!
-
-## Send Tokens
-
-```
-gaiacli send --amount=10faucetToken --chain-id=<name_of_testnet_chain> --name=<key_name> --to=<destination_address>
-```
+## Fund your account
 
-Note: The `--amount` flag accepts the format `--amount=<value|coin_name>`.
+The best way to get tokens is from the [Cosmos Testnet Faucet](https://faucetcosmos.network). If the faucet is not working for you, try asking [#cosmos-validators](https://riot.im/app/#/room/#cosmos-validators:matrix.org). The facuet needs the `cosmosaccaddr` from the key you wish to use for staking.
 
-Now, view the updated balances of the origin and destination accounts:
+After receiving tokens to your address, you can view your account's balance by typing:
 
+```bash
+gaiacli account <key_cosmosaccaddr>
 ```
-gaiacli account <origin_address>
-gaiacli account <destination_address>
-```
-
-You can also check your balance at a given block by using the `--block` flag:
 
-```
-gaiacli account <your_address> --block=<block_height>
-```
+> _*Note:*_ When you query an account balance with zero tokens, you will get this error: `No account with address <key_cosmosaccaddr> was found in the state.` This can also happen if you fund the account before your node has fully synced with the chain. These are both normal. Also, we're working on improving our error messages!
 
 ## Run a Validator Node
 
 [Validators](https://cosmos.network/validators) are responsible for committing new blocks to the blockchain through voting. A validator's stake is slashed if they become unavailable, double sign a transaction, or don't cast their votes. If you only want to run a full node, a VM in the cloud is fine. However, if you are want to become a validator for the Hub's `mainnet`, you should research hardened setups. Please read [Sentry Node Architecture](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#how-can-validators-protect-themselves-from-denial-of-service-attacks) to protect your node from DDOS and ensure high-availability. Also see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). There's also more info on our [website](https://cosmos.network/validators).
 
-Your `pubkey` can be used to create a new validator by staking tokens. You can find your validator pubkey by running:
+Your `cosmosvalpub` can be used to create a new validator by staking tokens. You can find your validator pubkey by running:
 
-```
+```bash
 gaiad tendermint show_validator
 ```
 
 Next, craft your `gaiacli stake create-validator` command:
 
-```
-gaiacli stake create-validator --amount=5steak --pubkey=<your_node_pubkey> --address-validator=<your_address> --moniker=satoshi --chain-id=<name_of_the_testnet_chain> --name=<key_name>
+> _*NOTE:*_  Don't use more `steak` thank you have! You can always get more by using the [Faucet](https://faucetcosmos.network/)!
+
+```bash
+gaiacli stake create-validator \
+  --amount=5steak \
+  --pubkey=$(gaiad tendermint show_validator) \
```

Comment:
would be nice if this was `--valpubkey` or `--valpub` instead of `--pubkey` CC @cwgoes @rigelrozanski

Reply:
++ but not in this PR